tools/clang/tools/dxil2spv/lib/dxil2spv.cpp

```diff
@@ -412,6 +413,88 @@ void Translator::createBinaryOpInstruction(llvm::BinaryOperator &instruction) {
   instructionMap[&instruction] = result;
 }
 
+void Translator::createHandleInstruction(llvm::CallInst &instruction) {
+  unsigned resourceClass =
+      cast<llvm::ConstantInt>(
+          instruction.getArgOperand(
+              hlsl::DXIL::OperandIndex::kCreateHandleResClassOpIdx))
+          ->getLimitedValue();
+  unsigned resourceRangeId =
+      cast<llvm::ConstantInt>(
+          instruction.getArgOperand(
+              hlsl::DXIL::OperandIndex::kCreateHandleResIDOpIdx))
+          ->getLimitedValue();
+  spirv::SpirvVariable *inputVar =
+      resourceMap[{resourceClass, resourceRangeId}];
+  if (!inputVar) {
+    emitError("No resource found corresponding to handle: %0", instruction);
+    return;
+  }
+
+  instructionMap[&instruction] = inputVar;
+}
+
+void Translator::createBufferLoadInstruction(llvm::CallInst &instruction) {
```

Comment:
nit: if you provided an example DXIL instruction and its translation result i.e., output SPIR-V as a comment, it would be easier to understand.

Reply:
Done